pkg/opsgenieIntegrationManager/client.go

```diff
@@ -0,0 +1,106 @@
+package opsgenieIntegrationManager
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"bitbucket.org/atlassianlabs/restclient"
+	"github.com/atlassian/voyager/pkg/util"
+	"github.com/atlassian/voyager/pkg/util/httputil"
+	"github.com/atlassian/voyager/pkg/util/pkiutil"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+)
+
+const (
+	asapAudience     = "micros-server"
+	asapSubject      = ""
+	integrationsPath = "/api/v1/opsgenie/integrations"
+)
+
+type Client struct {
+	logger     *zap.Logger
+	httpClient *http.Client
+	asap       pkiutil.ASAP
+	rm         *restclient.RequestMutator
+}
+
+func New(logger *zap.Logger, httpClient *http.Client, asap pkiutil.ASAP, baseURL *url.URL) *Client {
+	rm := restclient.NewRequestMutator(
+		restclient.BaseURL(baseURL.String()),
+	)
+	return &Client{
+		logger:     logger,
+		httpClient: httpClient,
+		asap:       asap,
+		rm:         rm,
+	}
+}
+
+// Gets OpsGenie integrations
+// return codes:
```

Comment:
These response codes are copy-pasted from Service Central client, please remove.

Reply:
The style was copied but the data is from the OG integration manager swagger. Do you still want these removed?

Reply:
Oh, ok.
> 201: Service created

Team created? or integrations created? 201 response code to GET request seems weird... Should be 200 I suppose?

Reply:
Ah right, that 201 was a copy-pasta error. Fixed these up.